cubic-server/world-storage/Level.cpp

```diff
@@ -0,0 +1,31 @@
+#include "Level.hpp"
+
+Level::Level()
+{
+}
+
+Level::~Level()
+{
+}
+
+void Level::addChunkColumn(_2d_pos pos, ChunkColumn &chunkColumn) {
+    _chunkColumns.insert({pos, chunkColumn});
+}
+
+void Level::addChunkColumn(_2d_pos pos) {
+    ChunkColumn chunkColumn;
+    _chunkColumns.insert({pos, chunkColumn});
```

Comment:
This will crash.
`_chunkColumns` is a map of `ChunkColumn &`, and the created `chunkColumn` will only be valid inside the `addChunkColumn` function.

Reply:
well done ! Thank you very much !

Reply:
Sadly this does not repair the segfault I have but it prevent a new one that will appear just after ^^

Reply:
It seems this is because of the hash ? I don't know, will check tomorrow.